dataactvalidator/config/sqlrules/d20_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,12 @@
+-- FundingAgencyCode is an optional field, but when provided must be a valid 3-digit CGAC agency code.
+SELECT
+    row_number,
+    funding_agency_code
+FROM detached_award_financial_assistance as dafa
+WHERE submission_id = {0}
+    AND funding_agency_code != ''
+    AND NOT EXISTS (
+        SELECT cgac_code
+        FROM cgac
```

Comment:
Stylistically, if you're using an alias on one table, you should be using that alias everywhere something from that table is referenced and you should be aliasing all other tables in the query as well for readability